tests/test_deltor.py

```diff
@@ -0,0 +1,273 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 Northwestern University.
+#
+# invenio-subjects-utils is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+from galter_subjects_utils.deltor import DeltasGenerator
+from galter_subjects_utils.scheme import Scheme
+from galter_subjects_utils.types_internal import Subject
+
+
+def subject_A():
```

Comment:
I simplified the test subjects now that it's clear that qualified/not qualified doesn't matter. I think the tests are clearer with the change too.